sig-windows/CONTRIBUTING.md

```diff
@@ -84,6 +85,34 @@ If you modifying an API in the SIG-Windows codebase, make sure you are aware of
 
 For the most up-to-date steps on how to build and run tests, please go to [https://github.com/kubernetes-sigs/windows-testing](https://github.com/kubernetes-sigs/windows-testing). It has everything you need to build and run tests, as well as links to the SIG-Windows configurations used on [TestGrid](https://testgrid.k8s.io/sig-windows).
 
+## Troubleshooting
+
+### Intro
+
+If you are having issues with network dependent services coming up with your Windows based container there is a workaround that may help.
+
+We can use the [Container Lifecycle Hooks](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/) to help ensure the service starts after the network is available.  Specifically we will be using the [PostStart](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks) hook.
+
+### Examples
+
+The first example will execute after the pod is up and in this particular case will restart the `dbconnect` service until `dbhost.example.com` can be resolved via DNS.  This command could also be modified to require said host to be reachable before exiting.
+
+```
+        lifecycle:
+          postStart:
+            exec:
+              command: ["powershell.exe","-command","do { Restart-Service -Name dbconnect; $Result = @(ping -n 1 dbhost.example.com) } while ( $Result -notcontains 'Approximate round trip times in milli-seconds:' )"]
```

Comment:
Nit: isn't checking `ping`'s exit code more robust than looking for a magical substring in the output?

Reply:
Actually, I wasn't able to see anything but `False` as the exit code no matter what happened.  Pinging something that resolves and something that doesn't produce the same exit code.  If someone know more about than I was able to find I'd be more than happy to change the example.

Reply:
Interesting....! Maybe worth mentioning in a comment then? So that admins who read this don't waste time trying to use exit codes?

Reply:
also make sure to comment that this script won't work in globalized operating systems with a different language.